dev_suites/dev_demo_ig_stu1/demo_suite.rb

```diff
@@ -73,7 +73,7 @@ class DemoSuite < Inferno::TestSuite
               token_url: 'https://example.com'
             }.to_json
 
-            output creds: creds
+            output creds:
```

Comment:
Ah.  TIL destructuring in Ruby 3.

Reply:
Yeah, it will take some getting used to, but it's nice.